lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+extern crate wasm_bindgen;
+#[macro_use] extern crate slice_as_array;
+
+use wasm_bindgen::prelude::*;
+use js_sys::*;
```

Comment:
We try not to import with `*` modules unless we use `prelude` or something similar. Maybe this is the correct usage here, but I would be more careful tbh.